app/views/home/_user_dropdown.html.erb

```diff
@@ -1,3 +1,5 @@
+<li id='notification_icon' class="drawer-pf-trigger dropdown"> </li>
+<%= mount_react_component('NotificationDrawerToggle', '#notification_icon', {:url => user_notifications_path}.to_json ) %>
```

Comment:
ditto, if user mgmt is turned off, we shouldn't probably show any notifications.